### PR TITLE
update PR to pull request

### PR DESCRIPTION
**Pull request by mattcosta7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,10 +48,10 @@ jobs:
 
 ##### Example Usage: Pull Requests labeled with `needs-review` and `size/XL`
 ```yaml
-name: Add needs-review and size/XL PRs to projects
+name: Add needs-review and size/XL pull requests to projects
 
 on:
-  pull_requests:
+  pull_request:
```
Comment (jayspadie) [inline README.md:54]: Weird that this isn't plural but `issues` above is.

**Review by jayspadie — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -48,10 +48,10 @@ jobs:
 
 ##### Example Usage: Pull Requests labeled with `needs-review` and `size/XL`
```
Comment (jayspadie) [inline README.md:49]: Tiny nit: Should probably user lowercase 'r' with "Pull requests labeled..."

Comment (jayspadie) [reply README.md:49]: meh... looks like we always capitalize it, so nevermind.

File: README.md (hunk 1 above)
Comment (mattcosta7) [reply README.md:54]: right? I checked it thrice

**Pull request merged**